client/src/components/TopBar.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useContext } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+
+import DragHandleIcon from '@material-ui/icons/DragHandle';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+
+import clsx from 'clsx';
+import { Link } from "react-router-dom";
+
+import UserContext from '../context/User'
+
+const useStyles = makeStyles((theme) => ({
+	root: {
+		flexGrow: 1,
+	},
+	menuButton: {
+		marginRight: theme.spacing(2),
+	},
+	title: {
+		flexGrow: 1,
+	},
+	AppBar: {
+		backgroundColor: '#fff',
+		color: '#000',
+	},
+}));
+
+
+
+export default function TopBar(props) {
+
+	const classes = useStyles();
+	const user = useContext(UserContext)
+	const [auth, setAuth] = useState(user.role);
+	const [anchorEl, setAnchorEl] = useState(null);
+	const open = Boolean(anchorEl);
+
+	const routes = ['chef']
+	const [top, setTop] = useState({
+		top: false,
+	});
+	const handleAuth = (event) => {
+		setAuth(event.target.checked);
+	};
+
+	const handleMenu = (event) => {
+		setAnchorEl(event.currentTarget);
+	};
+
+	const handleClose = () => {
+		setAnchorEl(null);
+	};
+
+	const toggleDrawer = (anchor, open) => (event) => {
+		if (event.type === 'keydown' && (event.key === 'Tab' || event.key === 'Shift')) {
+			return;
+		}
+
+		setTop({ ...top, 'top': open });
+	};
+
+
+	const list = (anchor) => (
+		<div
+			className={clsx(classes.list, {
+				[classes.fullList]: anchor === 'top' || anchor === 'bottom',
+			})}
+			role="presentation"
```

Comment:
Why is this role here?

Reply:
It was for something i was testing out I'll remove it

Reply:
removed